favourites.html

```diff
@@ -37,13 +37,15 @@
 
         <!-- App heading component -->
         <div class="item">
-          <h2 class="ui header">
-            <img
-              class="ui image"
-              src="./assets/images/travel-companion-logo.png"
-            />
-            <div class="content travel-title">Travel Companion</div>
-          </h2>
+          <a href="index.html">
```

Comment:
This is great Hakel :) Just one thing, since you removed the <div class="item"> it changes the position of the title in the navbar. Can you add that back? so just keep what you have here and wrap the extra div around it like this:

 <div class="item">
          <a href="index.html">
            <h2 class="ui header">
              <img
                class="ui image"
                src="./assets/images/travel-companion-logo.png"
              />
              <div class="content travel-title">Travel Companion</div>
            </h2>
          </a>
        </div>

Reply:
And the same on the results.html file :)